packages/glibc_standalone.rb

```diff
@@ -198,19 +79,28 @@ def self.build
     FileUtils.mkdir_p 'builddir'
     Dir.chdir('builddir') do
       File.write 'configparms', <<~EOF
-        slibdir=#{glibc_libdir}
-        rtlddir=#{glibc_libdir}
+        slibdir=#{CREW_GLIBC_PREFIX}
+        rtlddir=#{CREW_GLIBC_PREFIX}
         sbindir=#{CREW_PREFIX}/bin
         rootsbindir=#{CREW_PREFIX}/bin
       EOF
 
       system build_env.transform_keys(&:to_s), '../configure', *config_opts
       system "make PARALLELMFLAGS='-j #{CREW_NPROC}'"
     end
+
+    # compile crew-audit with system's glibc version, as we want it to work on system's glibc also
```

Comment:
This means we need to continue shipping a Glibc package (with Glibc headers) for the system Glibc on the machine?

Reply:
No, the older glibc package is only used to build the audit library. This ensures the compatibility with system's glibc runtime library (`/lib64/libc.so.6`)

Reply:
Right, but do we need to build a Glibc 2.39 package to build an audit library for new milestones running Glibc 2.39?